addons/mail/static/src/models/thread/thread.js

```diff
@@ -1711,10 +1828,28 @@ function factory(dependencies) {
                 'serverFoldState',
             ],
         }),
+        /**
+         * All messages ordered like they are displayed.
+         */
         orderedMessages: many2many('mail.message', {
             compute: '_computeOrderedMessages',
             dependencies: ['messages'],
         }),
+        /**
+         * Is transient trait for all messages ordered like they are displayed.
```

Comment:
I would remove even this comment, but it's not a big deal either way as long as the 2nd comment with task id is there.